amxmodx/amxmodx.cpp

```diff
@@ -1381,14 +1396,16 @@ static cell AMX_NATIVE_CALL show_menu(AMX *amx, cell *params) /* 3 param */
 	return 1;
 }
 
-static cell AMX_NATIVE_CALL register_plugin(AMX *amx, cell *params) /* 3 param */
+static cell AMX_NATIVE_CALL register_plugin(AMX *amx, cell *params) /* 5 param */
```

Comment:
Please, check arg count and make it backward compatible, so old plugins dont need to be recompiled

Reply:
new arguments are optional (see include)

Reply:
@KaidoRen This doesn't make it compatible with old plugins